Src/ILGPU/Static/TypeInformation.ttinclude

```diff
@@ -59,6 +59,8 @@ public class TypeInformation
 
     public bool IsFloat => Kind == TypeInformationKind.Float;
 
+    public bool IsCLSCompliant => !(IsUnsignedInt || Name == "Int8" || Name == "Half");
```

Comment:
Should the `Half` type be included here?

Reply:
I assumed it was not compliant, but I guess it's a struct, not a primitive. Removed.